Ladybird/HelperProcess.cpp

```diff
@@ -48,6 +48,8 @@ static ErrorOr<NonnullRefPtr<ClientType>> launch_server_process(
         if (!result.is_error()) {
             auto process = result.release_value();
 
+            process.client->set_pid(process.process.pid());
```

Comment:
instead of adding empty set_pid calls to the other clients, we could do an `if constexpr (requires { process.client->set_pid(pid_t{}) })` here (or whatever the right syntax is), since we're in a template function here.